### PR TITLE
Docs: replace `k6 Cloud` mention

### PR DESCRIPTION
**Pull request by ppcano**
k6 is now available as a managed service on Grafana Cloud. 

This is a small doc changes to remove the mention of `k6 Cloud`.

### CODE REVIEW

File: docs/sources/clients/k6/write-scenario.md [hunk 1]
```diff
@@ -55,14 +55,12 @@ These parameters can be adjusted in the load test:
...
-    k6 can be run locally, self-managed in a distributed way,
-    or it can be run highly-scalable within the k6 cloud.
+    k6 supports three [execution modes](https://k6.io/docs/get-started/running-k6/#execution-modes) to run a test: local, distributed, and cloud.
     Whereas running your k6 load test from a single (local
     or remote) machine is easy to set up and fine for smaller Loki clusters,
     the single machine does not load test large Loki installations,
     because it cannot create the data to saturate the write path.
-    Therefore, it makes sense to run
-    the tests in the [k6 Cloud](https://k6.io/cloud/).
+    For larger tests, consider [these optimizations](https://k6.io/docs/testing-guides/running-large-tests/), or run them in [Grafana Cloud k6](https://grafana.com/products/cloud/k6) or a Kubernetes cluster with [k6-operator](https://github.com/grafana/k6-operator).
```
Comment (JStickler) [inline docs/sources/clients/k6/write-scenario.md:63]: Changing the link from the name of the repo to the name of the Operator.
```suggestion
    For larger tests, consider [these optimizations](https://k6.io/docs/testing-guides/running-large-tests/), or run them in [Grafana Cloud k6](https://grafana.com/products/cloud/k6) or a Kubernetes cluster with the [k6-operator](https://github.com/grafana/k6 Operator).
```

**Review by JStickler — APPROVED** [review]
[docs team]  Just one small suggestion for rewording.

Thank you for contributing this update.

**Comment by JStickler** [comment]
@ppcano What are you trying to link to, the K6 product page or the K6 documentation?

**Comment by ppcano** [comment]
> @ppcano What are you trying to link to, the K6 product page or the K6 documentation?

Product page: https://grafana.com/products/cloud/k6/

**Comment by JStickler** [comment]
We've got this written up in the [Writer's Toolkit,](https://grafana.com/docs/writers-toolkit/writing-guide/references/) but it's not very clear.
I believe the format you want to use in this case is:

`[link text](/products/cloud/k6)`

Because the product docs are outside of the Loki repository, but still part of the Grafana website.

**Comment by ppcano** [comment]
@JStickler, it works. Thank you! 

I signed the CLA, so I guess the PR  is now ready for merging.

**Pull request merged**